biogtr/datasets/cell_tracking_dataset.py

```diff
@@ -0,0 +1,211 @@
+"""Module containing cell tracking challenge dataset."""
+from PIL import Image
+from biogtr.datasets import data_utils
+from biogtr.datasets.base_dataset import BaseDataset
+from scipy.ndimage import measurements
+from torch.utils.data import Dataset
+from torchvision.transforms import functional as tvf
+from typing import List
+import albumentations as A
+import glob
+import numpy as np
+import os
+import pandas as pd
+import random
+import torch
+
+
+class CellTrackingDataset(BaseDataset):
+    """Dataset for loading cell tracking challenge data."""
+
+    def __init__(
+        self,
+        raw_images: list[str],
+        gt_images: list[str],
+        padding: int = 5,
+        crop_size: int = 20,
+        chunk: bool = False,
+        clip_length: int = 10,
+        mode: str = "Train",
+        augmentations: dict = None,
+        gt_list: str = None,
+    ):
+        """Initialize CellTrackingDataset.
+
+        Args:
+            raw_images: paths to raw microscopy images
+            gt_images: paths to gt label images
+            source: file format of gt labels based on label generator
+            padding: amount of padding around object crops
+            crop_size: the size of the object crops
+            chunk: whether or not to chunk the dataset into batches
+            clip_length: the number of frames in each chunk
+            mode: `train` or `val`. Determines whether this dataset is used for
+                training or validation. Currently doesn't affect dataset logic
+            augmentations: An optional dict mapping augmentations to parameters. The keys
+                should map directly to augmentation classes in albumentations. Example:
+                    augs = {
+                        'Rotate': {'limit': [-90, 90]},
+                        'GaussianBlur': {'blur_limit': (3, 7), 'sigma_limit': 0},
+                        'RandomContrast': {'limit': 0.2}
+                    }
+            gt_list: An optional path to .txt file containing gt ids (cell
+                tracking challenge format)
```

Comment:
Ref to data format or just mention that `"track_id", "start_frame", "end_frame", "parent_id"` are expected columns?